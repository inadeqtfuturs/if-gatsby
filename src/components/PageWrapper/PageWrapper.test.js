import React from 'react';
import { PageWrapper } from '@components';
import ThemeRender from '@utils/testing';

describe('Main', () => {
  it('renders correctly', () => {
    const tree = ThemeRender(
      <PageWrapper>
        <p>hello world</p>
      </PageWrapper>
    );
    expect(tree).toMatchSnapshot();
  });
});
