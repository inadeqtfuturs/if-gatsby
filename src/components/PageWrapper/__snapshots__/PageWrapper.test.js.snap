// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `
<div
  className="sc-AxjAm gdTDOx"
>
  <p>
    hello world
  </p>
</div>
`;
